project/Dependencies.scala

```diff
@@ -49,7 +48,8 @@ object Dependencies {
     excludeGuavaJDK5("com.google.apis"                % "google-api-services-cloudbilling" % "v1-rev14-1.23.0"),
 
     "org.webjars"                    % "swagger-ui"          % "2.2.5",
-    "com.jason-goodwin"             %% "authentikat-jwt"     % "0.4.1",
+    "com.jason-goodwin"             %% "authentikat-jwt"     % "0.4.5"
```

Comment:
Is this because 0.4.1 is no longer compatible with new Jackson?

Reply:
no. I was seeing that sourceclear reported a jackson issue as a transitive dependency from authentikat-jwt. So I'm just being explicit here and making sure that authentikat-jwt doesn't pull in jackson - instead, it'll use the jackson I defined above.